vaadin-radio-button-flow-demo/src/main/java/com/vaadin/flow/component/radiobutton/demo/RadioButtonGroupView.java

```diff
@@ -239,6 +247,81 @@ private void themeVariantsHorizontal() {
         addCard("Theme variants", "Direction", horizontal, vertical);
     }
 
+    private void dataViewRefreshItem() {
+        // begin-source-example
+        // source-example-heading: Refresh Items
+        RadioButtonGroup<Employee> radioButtonGroup = new RadioButtonGroup<>();
+        radioButtonGroup.setLabel("Assignable Employees: ");
+        Employee employee1 = new Employee("Employee One", null);
+        Employee employee2 = new Employee("Employee Two", null);
+        Employee employee3 = new Employee("Employee Three", null);
+        RadioButtonGroupListDataView<Employee> dataView = radioButtonGroup
+                .setItems(employee1, employee2, employee3);
+        radioButtonGroup.setValue(employee3);
+        radioButtonGroup.addThemeVariants(RadioGroupVariant.LUMO_VERTICAL);
+
+        Button updateButton = new Button("Update second employee's name!",
+                click -> {
+                    employee2.setTitle("Employee 2");
+                    dataView.refreshItem(employee2);
+                });
+        // end-source-example
+
+        addCard(DATA_VIEW, "Refresh Items", radioButtonGroup,
+                updateButton);
+    }
+
+    private void dataViewAddItem() {
```

Comment:
It seems reasonable to me to add also a 'remove option' demo here, so as to show the users they can add/remove the items programmatically

Reply:
To remove which option? The selected one?

Reply:
No, selected one is a bit artificial here. I would propose remove the last one from the list. So the demo will add/remove to/from the end of list. Of course, you can then add the comment, that this is only for demo purpose, in a real life it would be possible choose any bean and pass it to remove method.